README.md

```diff
@@ -472,6 +487,163 @@ async function saveCoverageData (codeBlocks) {
 export default runUnitTests
 ```
 
+
+## running linter
+```js
+// runLinter.js
+import fs from 'fs'
+import saveToFile from './saveToFile'
+import standard from 'standard'
+import forEachCodeBlock from './forEachCodeBlock'
+
+export const paddingText = (width, string, padding = ' ') => {
+  return (width <= string.length) ? string : paddingText(width, string + padding, padding)
+}
+
+export const countLinterErrors = (results) => results[0].messages.length
+
+export const runStandardLinter = (contents, fix) => (
+  new Promise((resolve, reject) => {
+    standard.lintText(contents, { fix }, (err, { results }) => {
+      if (err) reject(err)
+      else resolve(results)
+    })
+  })
+)
+
+export const formatLineLinterError = (error) => (
+  paddingText(10, error.line + ':' + error.column + ': ') +
+  paddingText(20, error.filename) +
+  error.message + ' ' + '(' + error.ruleId + ')'
+)
+
+export const formatLinterError = (line, filename, output) => (error) => {
+  error.line += line - 1
+  error.filename = filename
+  error.fixedCode = output
+  return error
+}
+
+export const printLinterErrors = (errors) => {
+  console.error(errors.map(formatLineLinterError).join('\n'))
+}
+
+export const isFix = (options) => !!options._ && options._[0] === 'fix'
+
+export const insertCodeBlock = (code, filename) => {
+  const END = '`' + '`' + '`'
+  const BEGIN = END + 'js'
+  return [
+    BEGIN,
+    '// ' + filename,
+    code,
+    END
+  ].join('\n')
+}
+
+export const replaceAll = (text, pattern, replace) => text.split(pattern).join(replace)
+
+export const fixLinterErrors = async (errors, codeBlocks, targetPath = 'README.md') => {
+  let readme = fs.readFileSync(targetPath, 'utf8')
+  errors.map(({ filename, fixedCode }) => {
+    const code = codeBlocks[filename].contents.replace(/\n$/g, '')
+    if (fixedCode) {
+      readme = replaceAll(readme, insertCodeBlock(code, filename), insertCodeBlock(fixedCode, filename))
```

Comment:
It seems that this line is not covered by automated test.

Maybe, you can edit `example.md` to make linting fail, e.g. changing `a + b` to `a+b`. This should make the fixer fix the code.